pkgs/development/libraries/mp4v2/default.nix

```diff
@@ -25,8 +25,14 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   meta = {
+    description = "Stalled library provides functions to read, create, and modify mp4 files";
+    longDescription = ''
+      MP4v2 library provides an API to work with mp4 files
+      as defined by ISO-IEC:14496-1:2001 MPEG-4 Systems.
+      This container format is derived from Apple's QuickTime format.
+    '';
     homepage = https://code.google.com/archive/p/mp4v2/;
-    maintainers = [ ];
+    maintainers = [ Anton-Latukha ];
```

Comment:
Need to refer to `lib.maintainers` or somesuch here, otherwise this is an eval error.

Reply:
Yep. Such a maintainer.
I ran nox-review before adding maintainer field.

Reply:
What I mean is, there is no such binding in scope here, so eval'ing `meta` will fail.

Reply:
Thanks, I was too self-confident.